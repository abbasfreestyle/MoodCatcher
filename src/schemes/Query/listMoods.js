const listMoods = `
query {
  listMoods {
    items {
      id
      date
      feelings {
        items {
          id
          name
        }
      }
      mood
      comment
    }
  }
}`;

export default listMoods;
