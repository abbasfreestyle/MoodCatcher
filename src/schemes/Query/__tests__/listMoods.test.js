import listMoods from '../listMoods';

describe('Schemes > Query > List Moods', () => {
  test('should match the snapshot', () => {
    expect(listMoods).toMatchSnapshot();
  });
});
