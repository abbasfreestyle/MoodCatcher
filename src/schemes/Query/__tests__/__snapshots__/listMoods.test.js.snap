// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schemes > Query > List Moods should match the snapshot 1`] = `
"
query ($username: String!) {
  listMoods (filter: {
    username: {
      eq: $username
    }
  }) {
    items {
      id
      date
      feelings {
        items {
          id
          name
        }
      }
    mood
    comment
  }
}
}"
`;
